server/app/services/email/EmailSendClient.java

```diff
@@ -0,0 +1,25 @@
+package services.email;
+
+import com.google.common.collect.ImmutableList;
+
+/** Interface for sending emails from CiviForm. */
+public interface EmailSendClient {
+
+  /**
+   * Sends an email from CiviForm with one recipient.
+   *
+   * @param toAddress A single address of where the email should be sent.
+   * @param subject The email subject.
+   * @param bodyText The email body.
+   */
+  void send(String toAddress, String subject, String bodyText);
```

Comment:
Blocking: What happens if the email address is invalid?

Reply:
We don't strictly check for it, but it would likely cause an exception https://github.com/civiform/civiform/blob/e0f3efed9d4951a49e5df22818fb951c59204e1f/server/app/services/email/aws/SimpleEmail.java#L117-L121